avaframe/in3Utils/cfgUtils.py

```diff
@@ -681,6 +693,8 @@ def readAllConfigurationInfo(avaDir, specDir='', configCsvName='allConfiguration
             path to a directory where simulation configuration files can be found - optional
         configCsvName: str
             name of configuration csv file
+        searchCfgFiles: bool
+            if True search for available cfg files to check if sims already done: avaName/Outputs/com1DFA/configurationFiles
```

Comment:
Is this actually needed? Or should we keep it simpler and just always look for existing sims? I can't think of a relevant downside to this...

Reply:
where do we look for existing sims? so what is the indicator that a sim has been performed?  if for example exportData=False -no peakFiles are saved

Reply:
The same way we do it here? 
If no peakfiles exist: assume the simulation needs to be done again....